consts/consts.go

```diff
@@ -21,6 +21,7 @@ const (
 	// no more than 50 KiB of overhead but is likely much less)
 	NetworkSizeLimit = 2_044_723 // 1.95 MiB
 
+	// FIXME: should use the standard math.MaxUint8, etc.
```

Comment:
Fine for now, but let's avoid adding unrelated changes like this into larger PRs